### ISSUE TITLE
torch.sparse has no attribute softmax

### ISSUE THREAD

**Issue by InstantWindy** [haochenheheda/NFormer#7]
torch.sparse has no attribute softmax

**Comment by haochenheheda** [comment]
Please check this
https://pytorch.org/docs/stable/generated/torch.sparse.softmax.html

**Comment by InstantWindy** [comment]
My pytorch version is 1.4.0. There is no torch.sparse.softmax in this version.
